lib/rescue_registry/exception_handler.rb

```diff
@@ -10,6 +12,7 @@ def initialize(exception, **options)
 
       status = options[:status]
       if status == :passthrough
+        # FIXME: Won't work without Rails
         status = ActionDispatch::ExceptionWrapper.status_code_for_exception_without_rescue_registry(exception.class.name)
```

Comment:
Still need to figure out how to handle this. Not sure it makes sense to have this option outside of Rails. If that's the case then we should make an API for this that only applies in Rails.

Reply:
For now I've abstracted it a bit based on whether `ActionDispatch::ExceptionWrapper` is defined. I think there's probably further room for improvement here.